### DISCUSSION TITLE
Adding name to a job inside flow

### DISCUSSION THREAD

**Discussion by kumaranu** [Quantum-Accelerators/quacc#474] [General]
I have the following code where I add two jobs inside a `jobflow`.
```
from fireworks import LaunchPad
from jobflow.managers.fireworks import flow_to_workflow
from quacc.recipes.newtonnet.core import ts_job
from quacc.recipes.newtonnet.core import quasi_irc_job
from ase.io import read
from atomate.common.powerups import add_tags
import jobflow as jf

def add_to_lpad(index, atoms, ts_type=0, run=False):
    opt_swaps = {
        "max_steps": 200,
    }
    if ts_type == 0:
        job1 = jf.job(ts_job)(atoms, use_custom_hessian=True)
        job2 = jf.job(quasi_irc_job)(job1["atoms"])
    else:
        job1 = jf.job(ts_job)(atoms)
        job2 = jf.job(quasi_irc_job)(job1["atoms"])
    flow = jf.Flow([job1, job2])
    wf = flow_to_workflow(flow)
    LAUNCHPAD_LOC = '/global/home/users/kumaranu/fw_config/my_launchpad.yaml'
    lpad = LaunchPad.from_file(LAUNCHPAD_LOC)
    wf = add_tags(wf, {"class": f"sella_ts_prod_jun21_{ts_type}"})
    if run:
        lpad.add_wf(wf)

# indices = [f"000"]
indices = [f"{i:03}" for i in range(20)]

RUN=True

for index in indices:
    atoms = read(f'/global/home/users/kumaranu/Documents/sella_si/singlets/{index}.xyz')
    add_to_lpad(index, atoms, ts_type=0, run=RUN)
    add_to_lpad(index, atoms, ts_type=1, run=RUN)
```
I want to know how to add name-tags to both of these jobs? I would preferably want to add an "index" based tag with each job here.
Thanks.

**Comment by Andrew-S-Rosen** [comment]
Hi @kumaranu: I think what you're looking for is the `update_metadata` flag in Jobflow, as described [here](https://materialsproject.github.io/jobflow/jobflow.core.html#jobflow.core.flow.Flow.update_metadata) for flows and [here](https://materialsproject.github.io/jobflow/jobflow.core.html#jobflow.core.job.Job.update_metadata) for jobs.

So, that would look something like `job1.update_metadata({"tag": index})`. Does this answer your question?

**Reply by kumaranu** [reply]
Hi @arosen93,
Thanks for the quick response.
I tested my program with the update_metadata function on the jobs, it worked perfectly!
Thanks!

**Reply by Andrew-S-Rosen** [reply]
Great to hear!!
